python/eigenpy/__init__.py

```diff
@@ -14,5 +14,4 @@
 # eigenpy If not, see
 # <http://www.gnu.org/licenses/>.
 
-import numpy as np
 from eigenpy import *
```

Comment:
shouldn't it be `from .eigenpy import *` ?

Reply:
Yes, for Python3 you're right.